fplus-lib/src/core/mod.rs

```diff
@@ -3552,12 +3553,20 @@ _The initial issue can be edited in order to solve the request of the verifier.
             return Err(LDNError::Load("Last active allocation ID is active".into()));
         }
 
-        let total_requested: u64 = application_file.allocation.total_requested();
-        if let Some(total_requested_amount) = parse_size_to_bytes(&application_file.datacap.total_requested_amount.clone()) {
-            if total_requested == total_requested_amount {
-                return Err(LDNError::Load("Total datacap reached".into()));
-            }
-        }
+        let requested_so_far = application_file.allocation.total_requested();
+        let total_requested = parse_size_to_bytes(&application_file.datacap.total_requested_amount.clone()).ok_or(LDNError::Load("Can not parse total requested amount to bytes".into()))?;
+        
+        if requested_so_far == total_requested {
```

Comment:
```suggestion
        if requested_so_far >= total_requested {
```

Ultimately allocator can do whatever they want, so there's a risk they'll allocate more than the total amount requested initially.